test/e2e/network/conntrack.go

```diff
@@ -268,4 +269,46 @@ var _ = SIGDescribe("Conntrack", func() {
 			framework.Failf("Failed to connect to backend 2")
 		}
 	})
+
+	ginkgo.AfterEach(func() {
+		if ginkgo.CurrentGinkgoTestDescription().Failed {
+			// Dump conntrack table of each node for troubleshooting
+			getConntrackTable(fr, clientNodeInfo.name)
+			getConntrackTable(fr, serverNodeInfo.name)
+		}
+	})
 })
+
+// getConntrackTable dumps the node conntrack table
+func getConntrackTable(f *framework.Framework, nodeName string) {
+	// Create a pod to check the conntrack entries on the host node
+	privileged := true
+
+	pod := &v1.Pod{
```

Comment:
do we need to create a pod? Can't we reuse the already existing one (like kube-proxy)?

Reply:
Actually thinking more about it - maybe we can make the pod that we create priviliged and do that not in AfterEach, but rather before deleting the pod?

Reply:
BTW - it happens - it just happened to my on the gce presubmit...

Reply:
we need a image that has the `conntrack` userspace tool, 
I think agnhost does not have it, reusing kube-proxy pod will be great, 
how can I get the kube-proxy pod name for kind and gce? is there a consistent way to find them?

Reply:
you can probably list pods from kube-system namespace and use the kube-proxy regexp to match the name; not super nice but should work

Reply:
done